### ISSUE TITLE
SaberTrail on custom sabers is not removed

### ISSUE THREAD

**Issue by e00E** [Shadnix-was-taken/BeatSaber-SaberTailor#8]
I have disabled the trail and it works for the default saber.
However, when I use a custom saber then the trail shows up again.

**Comment by Shadnix-was-taken** [comment]
Yeah, I think as of right now, the trail settings will only work on default sabers and custom sabers which uses the default trails. Sabers with custom trails are probably not effected. I'll give it a look and see if I can change it.

Which set of custom sabers are you using? (link to ModelSaber or something like that would be nice)

**Comment by e00E** [comment]
Thank you for looking into it. The saber that still had a trail was this one https://modelsaber.com/Sabers/?id=1557988275 .
